tekton/release-pipeline-nightly.yaml

```diff
@@ -61,8 +61,24 @@ spec:
         inputs:
           - name: source
             resource: source-repo
-    - name: publish-images
+    - name: build-base-image
       runAfter: [build, unit-tests]
+      taskRef:
+        name: build-multiarch-base-image
+      params:
+        - name: pathToProject
+          value: $(params.package)
+        - name: imageRegistry
+          value: $(params.imageRegistry)
+      resources:
+        inputs:
+          - name: source
+            resource: source-repo
+        outputs:
+          - name: builtBaseImage
+            resource: builtBaseImage
+    - name: publish-images
+      runAfter: [build-base-image, build, unit-tests]
```

Comment:
NIT: since `build-base-image` already depends on `[build, unit-tests]`, this can depend on `build-base-image` alone.